pure-min.css

```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```

Comment:
Git sometimes does that to files. I deleted these lines whenever I saw them. That one slipped through.